R/data.table.R

```diff
@@ -2554,8 +2554,17 @@ setnames <- function(x,old,new) {
   invisible(x)
 }
 
-setcolorder <- function(x, neworder)
+setcolorder <- function(x, neworder=NULL)
 {
+  if (is.null(neworder)) {
+    keys = key(x)
+	if (is.null(indices(x))) {
```

Comment:
annoying, i know, but i think we settled on 2-space indentation for this project 😬

Reply:
annoying is when you are losing half of screen because of 4 space indentation :)